src/__stories__/__data__/bar-x/playground.ts

```diff
@@ -42,6 +42,14 @@ function prepareData(): ChartData {
                 ticks: {
                     pixelInterval: 120,
                 },
+                plotLines: [
```

Comment:
Will we have a possibility to render plotLines above the main chart?
![image](https://github.com/user-attachments/assets/86b3bc1e-fdee-4918-a487-c4dd75efa49a)